core/model/src/main/java/org/eclipse/sw360/antenna/model/artifact/facts/ArtifactSoftwareHeritageID.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.model.artifact.facts;
+
+import org.eclipse.sw360.antenna.model.artifact.ArtifactFact;
+import org.eclipse.sw360.antenna.model.artifact.ArtifactFactWithPayload;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ArtifactSoftwareHeritageID extends ArtifactFactWithPayload<String> {
+
+    public ArtifactSoftwareHeritageID(String softwareHeritageID) {
+        super(softwareHeritageID);
+    }
+
+    @Override
+    public String getFactContentName() {
+        return "Software Heritage URL";
+    }
+
+    @Override
+    public Class<? extends ArtifactFact> getKey() {
+        return ArtifactSoftwareHeritageID.class;
+    }
+
+    public static boolean isValid(String swhID) {
```

Comment:
currently this is optional to check. I would like to make that required, e.g. by the builder pattern.